SecurityManager/Clients.Web/res/Themes/NovaGray/Html/Style.css

```diff
@@ -236,6 +236,25 @@ table.accessControlEntryExpanded td.formGridTitleCell
   display: none;
 }
 
+div.mainContent > span.permissionsEditorValidationMessage,
+div.accessControlListContainer td.stateCombinationsContainer span.permissionsEditorValidationMessage
+{
+  display: flex;
+  gap: 0;
+  color: red;
+  margin-top: 0.5em;
+}
+
+div.mainContent > span.permissionsEditorValidationMessage::before,
+div.accessControlListContainer td.stateCombinationsContainer span.permissionsEditorValidationMessage::before
+{
+  content: '';
+  min-width: 16px;
```

Comment:
`min-width` is necessary to prevent overlap. 
This happens with `width`
![image](https://github.com/re-motion/Framework/assets/6972548/41174e77-7c71-46f6-ace9-bfe669f5fbbc)

Reply:
> We also set the background size here to 16px no-repeat so any bigger element than 16px would just introduce spacing?

It's the standard pattern based on the image width?